src/pbt/deployment/utils.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+
+def get_maven_opts():
+    """Get Maven options from environment with default fallbacks.
+
+    This function merges MAVEN_OPTS from environment variables with default values.
+    Environment variables take precedence, and missing defaults are appended.
+
+    Returns:
+        str: Merged Maven options string
+    """
+    default_opts = "-Xmx1024m -XX:MaxMetaspaceSize=512m -Xss32m"
+    existing_opts = os.environ.get("MAVEN_OPTS", "").strip()
+
+    if not existing_opts:
+        return default_opts
+
+    # Start with existing options, then add missing defaults
+    result_parts = existing_opts.split()
```

Comment:
No need to split this. Gives a false idea that string above can be extended to have more than mentioned variables